flake.nix

```diff
@@ -12,7 +13,7 @@
 
   outputs = inputs@{ self, nixpkgs, flake-parts, ... }:
     flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = nixpkgs.lib.systems.flakeExposed;
+      systems = import inputs.systems;
```

Comment:
@zimbatm Is the only reason we need an `import` here (rather than flake output itself being a systems list) is so we can support `path:` inputs?

Reply:
Actually flake output has to be attrset, so this would otherwise be `inputs.systems.systems`. Hmm.

Reply:
It's a trade-off. Path inputs make it easy for flake authors to maintain their own flake-specific list without creating a full sub-flake. And as you noted, it's shorter than `systems.systems` or `systems.lib.systems` if we want it to pass the `flake check` with no warning.